core/server/config/index.js

```diff
@@ -179,8 +192,14 @@ ConfigManager.prototype.set = function (config) {
             availableApps:    this._config.paths.availableApps || {},
             clientAssets:     path.join(corePath, '/built/assets/')
         },
+        scheduling: {
+            postScheduling: {
```

Comment:
the idea is to group all schedule configuration in one place.
`{ newsletter: { active: 'my-adapter' }, postScheduling: { active: 'pro-adapter' } }`